packages/salesforcedx-vscode-soql/src/messages/i18n.ja.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Japanese?

Reply:
I just copied what LWC had.

Reply:
Yes, we support Japanese in the other extensions. We work with a team that helps us keeping them up to date. Leave it as is and next time we engage with them they'll take care of updating this file too.